package.json

```diff
@@ -17,7 +17,8 @@
     "prepare": "husky install",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "cypress:headed": "cypress run --headed"
+    "cypress:headed": "cypress run --headed",
+    "update-types": "npx supabase gen types --lang=typescript --project-id $(grep -oP '(?<=^PROJECT_ID=).+' .env) > supabase/types.ts"
```

Comment:
`grep` is not cross environment friendly, and does this handle local / remote environments?

Reply:
Yeah that true, It'll be prefferable to write a script

Reply:
Using a script is fine but maybe overkill, you can just use `cross-env` and `dotenv` to load the variable within the command line (and it works on any environment). Also, would be nice to have a `local` and `remote` command to generate the types either locally with a local db or remotely targeting a deployed database.